cognite/client/credentials.py

```diff
@@ -288,7 +288,10 @@ def scopes(self) -> list[str]:
     def _refresh_access_token(self) -> tuple[str, float]:
         # First check if there is a serialized token cached on disk.
         if accounts := self.__app.get_accounts():
-            credentials = self.__app.acquire_token_silent(scopes=self.__scopes, account=accounts[0])
+            credentials = self.__app.acquire_token_silent_with_error(scopes=self.__scopes, account=accounts[0])
+            if "error" in credentials:
```

Comment:
Perhaps a retry loop with exponential backoff? There is some retryutil added recently

Reply:
Or maybe we should make this exception something the existing retry logic can.

Reply:
And I would also suggest being more specific about which error to retry. Is there more info in the error object?

Reply:
I see now this is only when using the persisted token, so I guess a single retry should suffice and that you want to do it regardless of the error mode. LGTM!

Reply:
But we should use the same for logic for the other login flows

Reply:
Done